internal/venice-client-common/src/main/java/com/linkedin/venice/controllerapi/D2ServiceDiscoveryResponseV2.java

```diff
@@ -3,31 +3,13 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 
 
+@Deprecated
```

Comment:
Just curious - why don't we just nuke it?

Reply:
It can cause `ClassNotFoundException` due to transitive dependencies if we remove it now. We need to adopt a proper practice to note deprecation versions and then remove deprecated classes in major